### PR TITLE
Cadl APIView tweaks for Stewardship board

### PR DESCRIPTION
**Pull request by tjprescott**
See issue: https://github.com/Azure/azure-sdk-tools/issues/4454

Example:
https://apiview.dev/Assemblies/Review/f9ffc96c223d4af4b77060886849c9bf/f6efa0c0e3e744efbad1af1386f62247?doc=True&diffOnly=False

### CODE REVIEW

File: tools/apiview/emitters/cadl-apiview/src/apiview.ts [hunk 1]
```diff
@@ -276,10 +274,21 @@ export class ApiView {
...
   stringLiteral(value: string) {
-    this.tokens.push({
-      Kind: ApiViewTokenKind.StringLiteral,
-      Value: `\u0022${value}\u0022`,
-    });
+    const lines = value.split("\n");
+    if (lines.length == 1) {
+      this.tokens.push({
+        Kind: ApiViewTokenKind.StringLiteral,
+        Value: `\u0022${value}\u0022`,
```
Comment (johanste) [inline tools/apiview/emitters/cadl-apiview/src/apiview.ts:281]: Do you really need the escape-of-quote here?

**Review by johanste — APPROVED** [review]

File: tools/apiview/emitters/cadl-apiview/src/apiview.ts (hunk 1 above)
Comment (tjprescott) [reply tools/apiview/emitters/cadl-apiview/src/apiview.ts:281]: Yes. Otherwise it won't render with quotes.

**Comment by tjprescott** [comment]
🥂

**Comment by tjprescott** [comment]
I will try to resolve the issue where my projection is not... projecting... before I merge this. Thanks for the review @johanste!!

**Comment by tjprescott** [comment]
It seems that the logic for filtering versions will work once APIView is released. Unfortunately I can't verify it beforehand due to a bug in the Cadl versioning library. I'll proceed with the release once the CI passes.

**Pull request merged**